x/pairing/keeper/helpers_test.go

```diff
@@ -84,6 +86,14 @@ func (ts *tester) addProviderExtra(
 	return nil
 }
 
+// addDelegators adds CONSUMER accounts without buying a subscription to be used as delegators
+func (ts *tester) addDelegators(count int) {
```

Comment:
why do we need this?
use AddAcounts instead

Reply:
done. see https://github.com/lavanet/lava/commit/4985b44d36d626bb5dcb72cd8a1fe821e723a21a